roles/confluent.test/molecule/rbac-scram-custom-rhel/molecule.yml

```diff
@@ -161,6 +161,7 @@ provisioner:
     group_vars:
       all:
         scenario_name: rbac-scram-custom-rhel
+        mask_sensitive_diff: "{{mask_sensitive_diff | default('true')}}"
```

Comment:
what task was it erroring out on? im curious why we dont have this issue w the mask_sensitive_logs var

Reply:
One of the taks `Copy CA Cert to Host` was marked for `mask_sensitive_diff` in confluent.test.ldap ( tls_custom_certs.yml) 
I have removed the flag and tests are fine now.